HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
+{
+    public string Id { get; set; }
+    public string UserName { get; set; }
+    public string Email { get; set; }
+    public string PasswordSalt { get; set; } 
```

Comment:
Just `Salt`

Reply:
do you mean to name it Salt instead of PasswordSalt or removing the other attribute (of course that doesn't feel right but I want to make sure!)